### PR TITLE
[Bug Fix] Re-use the alice fixture loader when loading fixtures

### PR DESCRIPTION
**Pull request by steffenbrem**
Fixes issue #45

### CODE REVIEW

**Comment by michalmarcinkowski** [comment]
Nice! Thank you Steffen! :+1:

**Comment by steffenbrem** [comment]
@michalmarcinkowski Np :) Thanks for merging so fast :+1:

File: src/ApiTestCase.php [hunk 1]
```diff
@@ -87,6 +92,12 @@ public function setUpDatabase()
         }
     }
 
+    public function setUp()
+    {
+        parent::setUp();
+        $this->fixtureLoader = new Loader();
```
Comment (lchrusciel) [inline src/ApiTestCase.php]: I would put it together with database setup or create setUpLoader method to keep consistency.

Comment (michalmarcinkowski) [inline src/ApiTestCase.php]: :+1: for `setUpLoader`, it has nothing to do with the database as we can use it with InMemoryRepository also.

Comment (lchrusciel) [inline src/ApiTestCase.php]: :+1: Good point, but still we should move it

**Pull request merged**